### PR TITLE
Build spark mllib tests with spark version

### PR DESCRIPTION
**Pull request by holdenk**

### CODE REVIEW

**Comment by jkbradley** [comment]
Thanks for pinging, will do.

File: lib/sparkperf/testsuites.py [hunk 1]
```diff
@@ -251,8 +251,8 @@ class MLlibTests(JVMPerfTestSuite, MLlibTestHelper):
     test_jar_path = "%s/mllib-tests/target/mllib-perf-tests-assembly.jar" % PROJ_DIR
 
     @classmethod
-    def build(cls):
-        run_cmd("cd %s/mllib-tests; %s clean assembly" % (PROJ_DIR, SBT_CMD))
+    def build(cls, spark_version):
+        run_cmd("cd %s/mllib-tests; %s -Dspark.version=%s.0 clean assembly" % (PROJ_DIR, SBT_CMD, spark_version))
```
Comment (jkbradley) [inline lib/sparkperf/testsuites.py]: This means we'll always use the initial release, not follow-up patch versions.  I'd prefer to use patch versions if it's not too much trouble.

Comment (holdenk) [inline lib/sparkperf/testsuites.py]: The current source code structure only differs for major versions for the testing code, are you planning on having different tests enabled for minor versions? This is only used at compile time.

Comment (feynmanliang) [inline lib/sparkperf/testsuites.py]: Well, we kind of hack our way through by having `MLLIB_SPARK_VERSION` be a float, so the change wouldn't be trivial...

I think this is fine for now, given that we have yet to add features which need perf testing in patch versions. If the need arises, adding a `MLLIB_SPARK_PATCH_VERSION` to `config.py` will not be too hard.

**Comment by feynmanliang** [comment]
LGTM

File: lib/sparkperf/testsuites.py (hunk 1 above)
Comment (jkbradley) [inline lib/sparkperf/testsuites.py]: OK sounds fine

**Comment by jkbradley** [comment]
@feynmanliang tried it out, so LGTM   Thanks!  I'll merge it.

**Pull request merged**